app/lib/identity_tijuana/member_sync.rb

```diff
@@ -24,7 +24,7 @@ def self.export_member(member, sync_id)
         sync(user, member, sync_type, sync_id, :export_member)
       rescue StandardError => e
         Rails.logger.error "[IdentityTijuana::export_member] failed: " \
-                           "#{e.message} member_id=#{member_id}" \
+                           "#{e.message} member_id=#{member.id}" \
```

Comment:
Would have been to to break this out as a seperate commit, but not worth doing now.

Also worth testing this log line by throwing an error in a mocked call to `sync()`, but doing so properly would duplicate some of the test changes in https://github.com/GetUp/identity-tijuana/pull/71 (in fact I may have already fixed that there) - so maybe lets just leave that there.

Reply:
yeah, I saw that you've fixed `member_id`, I'll remove the fix from this PR – it won't break the tests

Reply:
Moved this to a [separate commit](https://github.com/GetUp/identity-tijuana/pull/73/commits/c440d13a2cc98c72d6578c076e572faefbbc159e)